python/lsst/pipe/base/configOverrides.py

```diff
@@ -234,7 +236,8 @@ def applyTo(self, config):
 
         for otype, override in self._overrides:
             if otype is OverrideTypes.File:
-                config.load(override)
+                with override.open("r") as buffer:
```

Comment:
Now that `ResourcePath` is a standalone package we should consider using it in `pex_config` directly.